lib/yabeda/group.rb

```diff
@@ -9,6 +9,25 @@ class Group
 
     param :name
 
+    def tags
+      default_tags.keys
+    end
+
+    def default_tags
+      @default_tags ||= Concurrent::Hash.new
+
+      if name
+        Yabeda.groups[nil].default_tags.merge!(@default_tags)
+      else
+        @default_tags.dup
+      end
```

Comment:
While concept of implicit global group looks reasonable, such places to distinguish _global_ tags from _group-local_ ones are looking _very_ weird.

Reply:
I am not sure what you mean by 'such places?'